README.md

```diff
@@ -195,7 +196,7 @@ A list of Sketch plugins hosted at GitHub, in alphabetical order.
 - [Set Line Height](https://github.com/getflourish/sketch-set-line-height-plugin), by Florian Schulz: Plugin that allows you to set the line height of a text layer as a multiple of the font size.
 - [Setup Iconset Artboards](https://github.com/sebj/setup-iconset-artboards), by Seb Jachec: Creates artboards and export options necessary to export a .iconset in Sketch.
 - [Shapr](https://github.com/vincenzopetito/shapr), by Vincenzo Petito: A Sketch plugin to create element with the keyboards!
-- [Shared Text Styles](https://github.com/nilshoenson/shared-text-styles), by Nils Hoenson: A Sketch plugin that lets you import & export text styles throughout different Sketch documents.
+- [Shared Text Styles](https://github.com/nilshoenson/shared text styles), by Nils Hoenson: Easily import and export your Text Styles in Sketch.
```

Comment:
Why is this changed? Can you update your pull request so that it only affects your plugin?

Thanks! : )